tests/python/fake_bpy_module_test/fake_bpy_module_test/utils_test.py

```diff
@@ -114,7 +115,8 @@ def test_append_child(self):
         <return>
             <description>
             <data-type-list>
-""")
+""",
```

Comment:
Is this comma needed?

Reply:
No, but the Ruff formatter cannot be configured not to add it.

Reply:
I think adding a comma at the last of the parameters makes us difficult to distinguish between set and the function parameter.
How do you think about it?

Reply:
A comma is added automatically when there are more than one parameter.
The purpose of this rule is to minimize the diff in commits when adding or removing or switching parameters.
I am not bothered by it.
I prefer having an auto-formatter than not at all.
This allows focusing on code instead of formatting text manually.
This is really useful when format run on save.
Even if the style is less good.

Reply:
OK. I understood your idea.
However, adding comma is not intuitively for the reader perspective.
Is it possible to disable this format even if other commas (including list, set, and so on) do not work?

Reply:
There is no config for it.
The only way would be adding manually `# fmt: off` and `# fmt: on`.

Reply:
I think configuration file can select it like sphinx project.
https://github.com/sphinx-doc/sphinx/blob/master/.ruff.toml

Reply:
This is a linter rule not a formatter config.
The linter and formatter of Ruff are independent.
They do not influence each other.

Reply:
In my understanding, this PR targets to introduce Ruff linter not format the code.
Please consider to filter the error code by using the configuration file.

We could not accept this PR if it affects the readability.
It is important to keep the readability more than usefulness who does not introduce Ruff.

Reply:
This should be configurable from the editor.
I would like to filter the error to meet this project before merging this PR.

I found there is a format configuration. This doesn't work in your environment? 
https://github.com/astral-sh/ruff-vscode?tab=readme-ov-file#configuring-ruff

Reply:
This is a formatter configuration.
https://docs.astral.sh/ruff/formatter/#configuration
I think `skip-magic-trailing-comma` can solve this issue.
https://docs.astral.sh/ruff/settings/#format_skip-magic-trailing-comma

Maybe, we should consider other rules as well.